### PR TITLE
feat: promisify netLog.stopLogging()

### PR DESCRIPTION
**Pull request by miniak**
#### Description of Change

This PR promisifies `netLog.stopLogging()`

Depends on https://github.com/electron/electron/pull/16935

/cc @codebytere

#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes
Notes: Converted `netLog.stopLogging()` to return a Promise instead of taking a callback.

### CODE REVIEW

File: lib/browser/api/session.js [hunk 1]
```diff
@@ -2,22 +2,7 @@
...
-    session[wrappedSymbol] = true
-    const { cookies } = session
-    cookies.flushStore = deprecate.promisify(cookies.flushStore)
-    cookies.get = deprecate.promisify(cookies.get)
-    cookies.remove = deprecate.promisify(cookies.remove)
-    cookies.set = deprecate.promisify(cookies.set)
-  }
-  return session
-}
+const { fromPartition, Session, Cookies, NetLog, Protocol } = process.atomBinding('session')
```
Comment (codebytere) [inline lib/browser/api/session.js]: I recognize the usefulness of this change but can it perhaps be done in a follow-up PR to prevent scope creep?

Comment (codebytere) [inline lib/browser/api/session.js]: I recognize the usefulness of this change but can it perhaps be done in a follow-up PR to prevent scope creep?

Comment (miniak) [reply lib/browser/api/session.js]: Will do that

Comment (miniak) [reply lib/browser/api/session.js]: https://github.com/electron/electron/pull/16935

Comment (miniak) [reply lib/browser/api/session.js]: rebased on master, only one commit left in the PR

File: docs/api/net-log.md [hunk 2]
```diff
@@ -36,6 +35,16 @@ Starts recording network events to `path`.
 
 Stops recording network events. If not called, net logging will automatically end when app quits.
 
+**[Deprecated Soon](promisification.md)**
+
+### `netLog.stopLogging()`
+
+* `callback` Function (optional)
```
Comment (jkleinsc) [inline docs/api/net-log.md]: The promisified function doesn't accept a callback does it?

Comment (miniak) [reply docs/api/net-log.md]: @jkleinsc missed that, removed.

**Comment by jkleinsc** [comment]
Arm CI is failing on this PR for a reason unrelated to this PR.  #16992 will resolve that issue.

**Comment by miniak** [comment]
@jkleinsc can you please merge?

**Pull request merged**